installfiles/operator.yaml

```diff
@@ -8077,14 +8077,14 @@ spec:
         - name: API_SERVICE_SECURITY
           value: none
         - name: THUNDERNETES_SIDECAR_IMAGE
-          value: ghcr.io/playfab/thundernetes-sidecar-go:0.1.0
+          value: "ghcr.io/playfab/thundernetes-sidecar-go:0.1.0"

```

Comment:
should we check in this file? also, are quotes necessary?

Reply:
no, i don't think the quotes are necessary. we can revert this and the other set